graph-tutorial/views/calendar.hbs

```diff
@@ -1,5 +1,5 @@
-<!-- Copyright (c) Microsoft Corporation.
-     Licensed under the MIT License. -->
+// Copyright (c) Microsoft Corporation.
```

Comment:
I don't think this is valid comment syntax for Handlebars. Can you revert back to the HTML comment syntax?